docker-services.yml

```diff
@@ -64,7 +64,7 @@ services:
     ports:
       - "6379:6379"
   db:
-    image: postgres:9.6
+    image: postgres:12
```

Comment:
Why do you need this? What is the advantage of this new version. Moreover you do not have any mention in you commit message.

Reply:
To be up to date with postgress.